src/web/mjs/engine/DownloadManager.mjs

```diff
@@ -10,6 +10,7 @@ export default class DownloadManager extends EventTarget {
         super();
         this.queue = [];
         this.worker = setInterval( this.processQueue.bind( this ), 250 );
+        this.queuePaused = false;
```

Comment:
Personally i prefer not using flags/fields to control behavior, therefore i would clear the interval and unset the `this.worker` (e.g., to undefined) in case of pause and assign a new `this.worker` interval in case of resume.